inc/cropper/namespace.php

```diff
@@ -884,143 +896,46 @@ function massage_meta_data_for_orientation( array $meta_data ) {
 }
 
 /**
- * Filters 'img' elements in post content to add 'srcset' and 'sizes' attributes.
- *
- * @param string $content The raw post content to be filtered.
- * @param string $context The current filter context.
+ * Add our special handlers for width & height attrs and srcset attributes.
  *
- * @return string Converted content with 'srcset' and 'sizes' attributes added to images.
+ * @param string $filtered_image Full img tag with attributes that will replace the source img tag.
+ * @param string $context Additional context, like the current filter name or the function name from where this was called.
+ * @param int $attachment_id  The image attachment ID. May be 0 in case the image is not an attachment.
+ * @return string Full img tag with attributes that will replace the source img tag.
  */
-function filter_content_tags( string $content, string $context = null ) : string {
-	if ( null === $context ) {
-		$context = current_filter();
+function content_img_tag( string $filtered_image, string $context, int $attachment_id ) : string {
+	if ( strpos( $filtered_image, TACHYON_URL ) === false ) {
```

Comment:
Doesn't this create a hard requirement on tachyon? What if people use this plugin without it? 

The readme says:

> Some features in this plugin will work on their own however some are designed to augment the existing tools we use such as [Tachyon](https://github.com/humanmade/tachyon).

So I think let's check here if tachyon is on the system